pkg/kwokctl/k8s/feature_gates_data.sh

```diff
@@ -16,20 +16,32 @@
 # https://github.com/kubernetes/website/blob/main/content/en/docs/reference/command-line-tools-reference/feature-gates.md
 # Some details of the feature gate on the official website are not synchronized so here we use a script to get the real details in the code
 
-latest_release="${1:-24}"
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DIR="$(dirname "${BASH_SOURCE[0]}")"
+
+latest_release="${1:-}"
+minimum_release=6
```

Comment:
can you add a comment on why minimum_release is 6 here?

Reply:
Kubernetes added Feature Gate in 1.6

Reply:
Updated